org/allPRs.ts

```diff
@@ -1,6 +1,10 @@
 import { danger, warn, fail, GitHubCommit, markdown } from "danger"
-
 import yarn from "danger-plugin-yarn"
+import { ParseResult } from "./helpers/changelog/changelog-types"
+// @ts-ignore
+import { parsePRDescription } from "./helpers/changelog/parsePRDescription.js"
+// @ts-ignore
+import { changelogTemplateSections } from "./helpers/changelog/generateChangelogSectionTemplate.js"
```

Comment:
I think it might be better if I just create a new repo with these files to consume on both Eigen and here